submissions/shawn-derek/scripts/controllers/routes.js

```diff
@@ -0,0 +1,27 @@
+/* TODO:DONE Configure our two routes for this app with page.js,
+    by registering each URL your app can handle, linked to a
+    single controller function to handle it: */
+page('/', index);
+page('/about', about);
+page('/stats', stats);
+page('/new', newArticle);
+/* TODO:DONE What function do we call to activate page.js?
+    Fire it off now, to execute it: */
+page();
+
+function index(){
```

Comment:
Why are you making these one-line helper functions? Why not just put the right controller function in the page callback chain directly?